src/github.com/matrix-org/dendrite/clientapi/routing/routing.go

```diff
@@ -406,4 +406,31 @@ func Setup(
 			}}
 		}),
 	).Methods(http.MethodGet, http.MethodOptions)
+
+	r0mux.Handle("/user/{userId}/rooms/{roomId}/tags",
+		common.MakeAuthAPI("get_tag", authData, func(req *http.Request, device *authtypes.Device) util.JSONResponse {
+			// When a GET tag request is done, obtain all data of user from here and provide a JSON response.
+			// MakeAuthAPI will convert it to a HTTP response and reflect it on the browser.
```

Comment:
This comment is not required or should be added to `MakeAuthAPI` if it has a non-descriptive comment.